muttdown/main.py

```diff
@@ -160,10 +166,10 @@ def main():
     if args.print_message:
         print(rebuilt.as_string())
     elif args.sendmail_passthru:
-        cmd = c.sendmail.split() + ['-f', args.envelope_from] + args.addresses
+        cmd = c.sendmail.split() + ['-G', '-i', '-f', args.envelope_from] + args.addresses
```

Comment:
`-G` would need to be controlled by a config flag since it would only be appropriate when using this as a postfix tool